File: w24w09javalotto/src/main/java/kr/ac/kumoh/s20210463/w24w09javalotto/repository/LottoRepository.java
package kr.ac.kumoh.s20210463.w24w09javalotto.repository;

import kr.ac.kumoh.s20210463.w24w09javalotto.model.LottoNumber;

public interface LottoRepository {
    LottoNumber save(LottoNumber lottoNumber);
    LottoNumber find();
}

File: w24w07lotto/src/main/kotlin/kr/ac/kumoh/s20210463/w24w07lotto/repository/LottoRepository.kt
package kr.ac.kumoh.s20210463.w24w07lotto.repository

import kr.ac.kumoh.s20210463.w24w07lotto.model.LottoNumber

interface LottoRepository {
    fun save(lottoNumber: LottoNumber): LottoNumber
    fun find(): LottoNumber?
}